mozilla_django_oidc_db/registry.py

```diff
@@ -37,10 +37,10 @@ def decorator(plugin_cls: type[OIDCBasePlugin]) -> type[OIDCBasePlugin]:
 
         return decorator
 
-    def items(self) -> Iterable[Tuple[str, OIDCBasePlugin]]:
+    def items(self) -> Iterable[Tuple[str, OIDCPlugin]]:
```

Comment:
```suggestion
    def items(self) -> Iterable[tuple[str, OIDCPlugin]]:
```
these kind of conversions can typically be fixed automatically with [ruff rules](https://docs.astral.sh/ruff/rules/#pyupgrade-up)

a future PR could do the switch to Ruff :)